src/Components/Task.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+
+const Task = ({ description, setTaskList, tasks, id, setTaskToBeEdited, setNewDescription }) => {
+    const [isStrikeThrough, setStrikeThrough] = useState(false);
+
+    const removeTask = (id) => {
+        return setTaskList(tasks.filter(task => task.id !== id))
+    };
+
+    useEffect(() => {
+        const modal = document.querySelector('.modal');
+        const overlay = document.querySelector('.overlay');
```

Comment:
`document.querySelector` is used in Vanilla JS. When using React we try to avoid this. Use `refs` instead if you want to get the reference to a DOM element. Please refer this link [https://reactjs.org/docs/hooks-reference.html#useref](https://reactjs.org/docs/hooks-reference.html#useref)

Reply:
Deleted the modal entirely after merging TaskEditor and TaskAdder. So, did not need to use refs.